core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -185,6 +185,23 @@ angular.module('oppia').directive('stateInteractionEditor', [
             _updateInteractionPreviewAndAnswerChoices();
           };
 
+          var updateMultiChoiceInputAnswerGroupsOnDelete =
+            function(deletedIndexes) {
+              var answerGroups = ResponsesService.getAnswerGroups();
+              deletedIndexes.forEach( function(deletedIndex) {
```

Comment:
Drop space before function. (Follow existing style.)

Reply:
done

Reply:
Not done.